qiskit/transpiler/passes/utils/filter_op_nodes.py

```diff
@@ -11,8 +11,9 @@
 # that they have been altered from the originals.
 
 """Filter ops from a circuit"""
+from __future__ import annotations
 
-from typing import Callable
+from collections.abc import Callable
```

Comment:
Is this a necessary change or just to normalize all the `import Callable`?

Reply:
Using imports from `typing` is [deprecated](https://peps.python.org/pep-0585/#implementation)